internal/provider/sdkv2/provider.go

```diff
@@ -364,6 +387,22 @@ func (p *sdkProvider) configure(ctx context.Context, d *schema.ResourceData) (an
 		config.RetryMode = mode
 	}
 
+	if config.ClientCertificate == "" {
+		if v := os.Getenv("AWS_CLIENT_CERTIFICATE"); v != "" {
+			config.ClientCertificate = v
+		}
+	}
+	if config.ClientPrivateKey == "" {
+		if v := os.Getenv("AWS_CLIENT_PRIVATE_KEY"); v != "" {
+			config.ClientPrivateKey = v
+		}
+	}
+	if config.ClientPrivateKeyPassphrase == "" {
+		if v := os.Getenv("AWS_CLIENT_PRIVATE_KEY_PASSPHRASE"); v != "" {
```

Comment:
Could you please rename this environment variable to TF_AWS_CLIENT_PRIVATE_KEY_PASSPHRASE to avoid potential clash with environment variables that Amazon define.